utilities/decrypt_data/decrypt.py

```
@@ -0,0 +1,90 @@
+import subprocess
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)